backend/backend_daemon.py

```diff
@@ -491,6 +508,19 @@ def sync_user_keys_of_vessels():
 
   log.info("[sync_user_keys_of_vessels] thread started.")
 
+  # Beginning of part of fix for Issue #152
+  # If we are in a django version that has django.setup() (1.7+?), run
+  # django.setup(). This is needed for django compatibility. So that it works,
+  # undo the repy custom patching of the type builtin so that django can use it.
+  # We'll turn it back on after.
+  if hasattr(django, 'setup'):
+    temp_type = __builtins__.type
+    import safe
+    __builtins__.type = safe._type
+    django.setup()
+    __builtins__.type = temp_type
+  # end of part of fix for Issue #152
+
```

Comment:
See above --- we require a suitable Django version with `django.setup` from now on.